iris/src/iris/pipeline/shared/transcription/youtube_utils.py

```
@@ -0,0 +1,170 @@
+        "--merge-output-format",
+        "mp4",
+        "--no-warnings",
+        "--no-playlist",
+        "-o",
+        str(output_path),
+        "--",
+        url,
+    ]
+    logger.info("Downloading YouTube video %s -> %s", url, output_path)
+    try:
+        subprocess.run(  # nosec B603
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)